packages/lib-classifier/src/store/tasks/DrawingTask.js

```diff
@@ -1,16 +1,22 @@
 import { types } from 'mobx-state-tree'
 import Task from './Task'
+import { Line, Point } from './drawingTools'
 
 // TODO: Need to define tool models
 
 const Drawing = types.model('Drawing', {
-  answers: types.array(types.frozen({
-    label: types.string,
-    next: types.maybe(types.string)
-  })),
   help: types.optional(types.string, ''),
   instruction: types.maybe(types.string),
-  tools: types.array(types.frozen()),
+  tools: types.array(types.union({
+    dispatcher: (snapshot) => {
```

Comment:
yarn lint detects an error on line 11: `11:28  error  Expected to return a value at the end of arrow function 'dispatcher'`.

Adding a `default` clause that returns `null` should resolve the error.